backend/app/routes/admin/putAdmin.js

```diff
@@ -0,0 +1,28 @@
+const to = require('await-to-js').default;
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const { firstName, lastName, contact, username } = req.body;
+  const userRecord = await Admin.findOne({
+    email: res.locals.decode.email,
+  });
+
+  try {
```

Comment:
you can omit try catch here, you can directly use await-to-js and check for error's if returned any